### DISCUSSION TITLE
I am using Keycloak version 17.0.1 and I want to access the Keycloak Admin REST API documentation in that version.

### DISCUSSION THREAD

**Discussion by chelzeasalvador-ubidy** [keycloak/keycloak#27016] [Q&A]
https://www.keycloak.org/docs-api/17.0.1/rest-api/index.html

**Comment by vmuzikar** [comment]
Please consider upgrading to the latest Keycloak version. 17 has been unsupported for long time now and is also exposed to security vulnerabilities.
